### PR TITLE
{lib}[foss/2020a] igraph v0.8.2, leidenalg v0.8.2, PyCairo v1.18.2, ... w/ Python 2.7.18 + 3.8.2

### PR DESCRIPTION
**Pull request by zarybnicky**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }),
+    (name, version, {
+        'buildopts': '--use-pkg-config',
+        'installopts': '--use-pkg-config',
+        'testopts': '--use-pkg-config',
+        'checksums': ['02cc8e87d2000ea1da60e9efc9287764fe81e445ab9850ec728b006c37620289'],
+    }),
+]
+
+runtest = 'python setup.py test'
```
Comment (boegel) [inline easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb:41]: This should be listed above, for `leidenalg` specifically?

```
    'runtest': 'python setup.py test',
```

File: easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('igraph', '0.8.2'),
+    ('python-igraph', '0.8.0'),
+    ('PyYAML', '5.3'),  # Test-only
```
Comment (boegel) [inline easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb:21]: Should be a build dep if it's test-only?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    # Test-only dependency
+    ('ddt', '1.4.1', {
+        'checksums': ['0595e70d074e5777771a45709e99e9d215552fb1076443a25fad6b23d8bf38da'],
+    }),
+    (name, version, {
+        'buildopts': '--use-pkg-config',
```
Comment (smoors) [inline easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb:34]: @zarybnicky the build step does not do anything when installing with pip (building occurs as part of the install step), so better to remove the buildopts here.

**Review by smoors — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2451.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/71d65c9d5a5c43439b9c68f39d0b974e for a full test report.

File: easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb:34]: @smoors `use_pip` is set to `False`, but `python setup.py build` is being run in the build step

**Review by boegel — APPROVED** [review]

File: easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb (hunk 3 above)
Comment (smoors) [reply easybuild/easyconfigs/l/leidenalg/leidenalg-0.8.2-foss-2020a-Python-3.8.2.eb:34]: ah I missed that it was set to False. you can ignore my comment then.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @zarybnicky!

**Pull request merged**